lib/WebpackConfig.js

```diff
@@ -243,6 +245,16 @@ class WebpackConfig {
         this.tsConfigurationCallback = callback;
     }
 
+    enableForkedTypeScriptTypesChecking(forkedTypeScriptTypesCheckOptions = {}) {
+
+        if (!this.useTypeScriptLoader) {
+            throw new Error('TypeScript loader must be enabled. Call `enableTypeScriptLoader()` first.');
+        }
```

Comment:
I can't remember if I made this comment before or not, but checks like this need to be moved to be during the config generation process. It should be fine to call `enableTypeScriptLoader()` and `enableForkedTypeScriptTypesChecking()` in whatever order.